src/app/(home)/globe/index.tsx

```diff
@@ -48,11 +48,11 @@ function GlobeInner() {
   }, []);
 
   // Update the URL when necessary
-  useEffect(() => {
-    if (!pageIsMounting) {
-      router.push(pageUrl);
-    }
-  }, [router, pageUrl, pageIsMounting]);
+  // useEffect(() => {
+  //   if (!pageIsMounting) {
+  //     router.push(pageUrl);
+  //   }
+  // }, [router, pageUrl, pageIsMounting]);
```

Comment:
@vgeorge I'm not sure why this exists but it doesn't seem to break anything if removed. But it does interfere with NextJS routing, it redirects to `/` every time we try to go to the search page.

Reply:
@oliverroick this was the previous implementation of URL navigation for areas and can be safely removed.